src/App.js

```diff
@@ -1,23 +1,90 @@
-import logo from './logo.svg';
-import './App.css';
+import { push, getDatabase, ref, onValue, remove } from 'firebase/database';
```

Comment:
Make sure to add a customized favicon in the public folder and website title/description in the index.html